.github/workflows/build-test-package.yml

```diff
@@ -157,6 +157,18 @@ jobs:
     - name: 'Build 🐍 Python 📦 package'
       run: |
         export ITK_PACKAGE_VERSION=${{ env.itk-wheel-tag }}
+        pwd
+        git clone https://github.com/InsightSoftwareConsortium/ITKMeshToPolyData.git
```

Comment:
Do we need to change ITKUltrasound to add this there? It will now indirectly depend on `ITKMeshToPolyData`.

Reply:
Yes, ITKUltrasound will need to be updated with the ITKMeshToPolyData dependency. I'm not aware of any other external modules depending on ITKBSplineGradient, but if there are then they should be updated too.